pkg/key/app.go

```diff
@@ -203,7 +204,7 @@ func UserSecretNamespace(customResource v1alpha1.App) string {
 }
 
 func Version(customResource v1alpha1.App) string {
-	return customResource.Spec.Version
+	return strings.TrimPrefix(customResource.Spec.Version, "v")
```

Comment:
Same here. Can you add a comment on why we added this.

Reply:
done